classes/class-wc-connect-shipping-label.php

```diff
@@ -110,6 +110,23 @@ protected function get_packages( WC_Order $order ) {
 			return $formatted_packages;
 		}
 
+		protected function get_all_packages() {
+			$packages = $this->settings_store->get_packages();
+
+			if ( ! $packages ) {
+				return new stdClass();
```

Comment:
The empty array will be serialized to json's `[]` which will lead to a React error, because it expects an object. Not sure if there's a way of working around that.